products/node-feature-discovery.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Kubernetes Node Feature Discovery
+category: server-app
+tags: kubernetes
+iconSlug: kubernetes
+permalink: /kubernetes-node-feature-discovery
+alternate_urls:
+-   /k8s-node-feature-discovery
+-   /node-feature-discovery
+versionCommand: |-
+  # Retrieve the image, along with the tag - which all point to the same version, post deployment on K8s
+  kubectl get pods --all-namespaces -o jsonpath="{.items[*].spec['initContainers', 'containers'][*].image}" |\
+  tr -s '[[:space:]]' '\n' |\
+  sort |\
+  uniq -c
```

Comment:
Wouldn't this show images for all containers instead of just for node-feature-discovery?

Reply:
I've updated it to exec into a deployment and print the version out.